Remotion/Web/Development.WebTesting.IntegrationTests/ChromeBinariesProviderTest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.IO;
+using System.Text.RegularExpressions;
+using NUnit.Framework;
+using Remotion.Web.Development.WebTesting.WebDriver.Configuration.Chrome;
+
+namespace Remotion.Web.Development.WebTesting.IntegrationTests
+{
+  [TestFixture]
+  public class ChromeBinariesProviderTest
+  {
+    [Test]
+    public void GetInstalledExecutable ()
+    {
+      var provider = new ChromeBinariesProvider();
+
+      var result = provider.GetInstalledExecutable();
+
+      Assert.That (result, Is.Not.Null);
+      Assert.That (result.BrowserBinaryPath, Is.EqualTo (@"C:\Program Files (x86)\Google\Chrome\Application\chrome.exe"));
+      Assert.That (
+          result.DriverBinaryPath,
+          Is.StringMatching (
+              Regex.Escape (Path.GetTempPath()) + @"Remotion.Web.Development.WebTesting.WebDriver\\chromedriver\\chromedriver_v\d+\.\d+\.\d+\.\d+\\chromedriver.exe"));
+      Assert.That (result.UserDirectory, Is.Not.Null.Or.Empty);

```

Comment:
We could assert the format of `UserDirectory` like in the assert above with `DriverBinaryPath`

Reply:
Fixed in https://github.com/re-motion/Trunk-SVN2Git/pull/12/commits/cc21cb58b42c8d277dc5010ce41de8afefffb1e3.